lib/main.dart

```diff
@@ -114,42 +103,44 @@ class _MyHomePageState extends State<MyHomePage> {
   ];
 
   Future<void> _showChoiceDialog(BuildContext context) {
-    return showDialog(context: context, builder: (BuildContext context) {
-      return AlertDialog(
-        title: const Center(child: Text('Make a Choice...')),
-        content: SingleChildScrollView(
-          child: Row(
-            mainAxisAlignment: MainAxisAlignment.spaceAround,
-            children: <Widget>[
-              ElevatedButton(
-                  onPressed: () => pickImageFromGallery(),
-                  child: Icon(Icons.photo_album_outlined),
-                  style: ButtonStyle(
-                    shape: MaterialStateProperty.all(CircleBorder()),
-                    padding: MaterialStateProperty.all(EdgeInsets.all(10)),
-                    backgroundColor: MaterialStateProperty.all(Colors.cyan),
-                    overlayColor: MaterialStateProperty.resolveWith<Color?>((states) {
-                      if (states.contains(MaterialState.pressed)) return Colors.red;
-                    }),
+    return showDialog(
+        context: context,
+        builder: (BuildContext context) {
+          return AlertDialog(
+            title: const Center(child: Text('Make a Choice...')),
+            content: SingleChildScrollView(
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceAround,
+                children: <Widget>[
+                  ElevatedButton(
+                    onPressed: () => {}, //pickImageFromGallery(),
```

Comment:
uncorrect arrow function. These have some options:
1. `onPressed: () => pickImageFromGallery()` 
2. `onPressed: pickImageFromGallery`
3. `onPressed: () {
pickImageFromGallery();
}`
4. `onPressed: () {}`

actually, try to minimize your code as possible where it is applicable. For this case 4th is good. But if you wanna you that pick method - you should use the 2nd, depends on pick method, do you want to pass some arguments there or not